src/providers/apolloClient.tsx

```diff
@@ -124,10 +125,37 @@ export const cache = new InMemoryCache({
   },
 });
 
+const retryLink = new RetryLink({
+  delay: {
+    initial: 800, // Wait 800ms before the first retry after a failed request
+    max: 1200, // Maximum wait time between retries
+    jitter: true,
+  },
+  attempts: {
+    max: 3,
+    retryIf: (error: CustomFetchNetworkError, operation) => {
+      // Only retry operations that seem to be queries, like "GetWidgets"
+      // Avoid retries on mutations as they may not be idempotent
+      if (!operation?.operationName?.match(/^(Get)/)) {
+        return false;
+      }
+      switch (error?.statusCode) {
+        case 401:
+        case 403:
+        case 404:
+        case 500:
+          return false;
+      }
+      return !!error;
+    },
+  },
+});
+
 const apolloClient = new ApolloClient({
   link: from([
     apolloErrorLink, // handle errors
     sentryLink, // instrument graphql requests for sentry
+    retryLink,
```

Comment:
with this placement, if a request fails 2 times and succeeds on the 3rd try, we will not get a Sentry error at all, right?

Reply:
Alright, I think that is good 👍